homeassistant/components/ps4/media_player.py

```diff
@@ -5,7 +5,13 @@
 from pyps4_2ndscreen.errors import NotReady, PSDataIncomplete
 import pyps4_2ndscreen.ps4 as pyps4
 
-from homeassistant.components.media_player import ENTITY_IMAGE_URL, MediaPlayerDevice
+from homeassistant.components.media_player import (
+    ENTITY_IMAGE_URL,
+    STATE_IDLE,
```

Comment:
The `STATE_*` are actually defined in `homeassistant.const` and imported by `media_player`.  Revert this change to import them from the defining module.

Reply:
Sounds good. Reverted.